README.md

```diff
@@ -191,6 +192,31 @@ where they will briefly update the limits. After you get confirmation that they
 
 ---
 
+# AltLayer MACH
+
+## Operator Flow
+
+1. generate and encrypt a BLS keystore using the [EigenLayer CLI](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation#create-and-list-keys)
+2. Determine which `quorums` and `socket` you wish to register for
+3. Sign digest establishing ownership of your newly generated BLS key
+
+           ./avs-cli altlayer prepare-registration --operator-id {operator_id} --bls-keystore {path_to_keystore} --bls-password {password} --quorums {0,1} --socket {socket}
+
+4. Send the result of the previous command to the ether.fi team via `restaking@ether.fi`
+5. Wait for confirmation from the ether.fi team that your registration is complete
+6. Follow the [AltLayer Operator Guide](https://docs.altlayer.io/altlayer-documentation/altlayer-facilitated-actively-validated-services/altlayer-mach-avs/operator-guide) skipping the `Opt-in and out of MACH AVS` sections. The `Opt-in` was already handling by the steps above. The `OPERATOR_ECDSA_ADDRESS` is `ecdsaSigner` of ether.fi operator contract you are registering with.
```

Comment:
Will the node operators know to go to etherscan and read the `ecdsaSigner` value for the correct contract or do I need to include more guidance?

Reply:
that seems like a really weird decision. Are you sure the altlayer team intends that value to be set to the value of the 1271-signer?

Reply:
seems so

<img width="276" alt="Screenshot 2024-08-18 at 3 32 46 PM" src="https://github.com/user-attachments/assets/a4603f50-b7f6-4a98-b002-068436d2ecfc">